UnrealPlugin/README.md

```diff
@@ -216,6 +216,41 @@ a dedicated server, so you could wrap the call to ReadyForPlayers() as such at t
 #endif
 ```
 ----
+##### Set the game server port number with MPS's configuration.
+```cpp
+    FGameServerConnectionInfo ConnectionInfo = UGSDKUtils::GetGameServerConnectionInfo();
+
+    for (auto& GamePorts : ConnectionInfo.GamePortsConfiguration) {
```

Comment:
wouldnt it make a lot more sense to integrate this directly into the plugin, specifically below [GSDKInternal.cpp line 81](https://github.com/PlayFab/gsdk/blob/master/UnrealPlugin/Source/PlayfabGSDK/Private/GSDKInternal.cpp#L81)? The <portname> could then be a string hardcoded, and the documentation could state that the port mapping for the UE4 port in the MPS configuration should be called for example "ue4_port"

Reply:
Ok, I've learned enough to chime into this conversation meaningfully.

1. I'll apply the changes we agree on in a separate PR, since this one is now a merge conflict with my documentation overhaul. I'll close this PR once the conversation is complete.
1. I see both points.
    1. @stkrwork suggests that if the name of the port is fixed, then this block can be moved into the GSDK itself, and the customer doesn't have to copy-pasta this boilerplate code. I think this is a very valid suggestion.
    1. @raymondchams is suggesting that the port name is dev-controlled in the Game Manager, and the customer can define multiple ports, and having direct control allows the customer more flexibility when defining multiple ports. I definitely see how a hidden-requirement/magic-required-port-name could be the source of its own frustration.
    1. I think my opinion is that both are satisfied "well enough" with a sufficiently specific and unique name, and good debug logging, such that with a unique name it won't overlap other customer scenarios. I also think that the less code the customer writes, and the more it "just works", the better. With that in mind, I propose a fixed name, renamed: "UnrealServerGsdkHostPort". The customer will have to define a port in Game Manager with that exact name. Doing so will just be an extra requirement for Unreal servers that use this GDSK plugin. And we'll describe that step in an appropriate place. This code will be moved into GSDK initialization as strwork has suggested.
    1. I'm going to move forward with my suggestion asap, barring additional resistance.